test/functional/feature_segwit.py

```diff
@@ -44,6 +44,7 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
+    assert_not_equal,
     assert_greater_than_or_equal,
     assert_is_hex_string,
```

Comment:
Nit. This file has alphabetical imports.

```suggestion
    assert_greater_than_or_equal,
    assert_is_hex_string,
    assert_not_equal,
```